modules/react/src/components/export-video/export-video-panel-preview.js

```diff
@@ -148,46 +150,35 @@ export class ExportVideoPanelPreview extends Component {
     // Adds mapbox layer to modal
     const map = this.mapRef.current.getMap();
     const deck = this.deckRef.current.deck;
+
+    const layerOrder = this.props.mapData.visState.layerOrder;
+    const keplerLayers = layerOrder
```

Comment:
Since this block of code is the same as what is in render(), you should create a function on the class that returns `keplerLayers` and reuse.

Reply:
Good idea! Implemented